pages/common/octave.md

```diff
@@ -0,0 +1,28 @@
+# octave
+
+> GNU Octave is a programming language for scientific computing.
+> More information: <https://docs.octave.org/latest/Invoking-Octave-from-the-Command-Line.html>.
+
+- Start an interactive Octave session:
```

Comment:
```suggestion
- Start an interactive session:
```

There is no need to mention that this is an Octave session because the page description already tells us that this page is for Octave. 😄